modules/utility/init.zsh

```diff
@@ -6,7 +6,8 @@
 # Colours
 #
 
-if (( ${terminfo[colors]} >= 8 )); then
```

Comment:
I tried it like this first, but if $terminfo[colors] is empty, zsh tells me that i can't change a readonly variable, like in this example:

`
$ readonly TEST
$ echo $TEST

$ echo ${TEST:=0}
zsh: read-only variable: TEST
`
Sadly i see no way around using a temporary variable :(